dirty_cat/similarity_encoder.py

```diff
@@ -273,6 +283,28 @@ def fit(self, X, y=None):
         self
         """
 
+        if self.handle_missing not in ['error', '']:
+            template = ("handle_missing should be either \'error\' or "
```

Comment:
Slight detail: you don't need to protect the ' when the string is with doublequotes: "a 'b' c" is valid.
```suggestion
            template = ("handle_missing should be either 'error' or "
```